interview-questions.md

```diff
@@ -8,36 +8,103 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Foreign keys are used to pull information between tables in databases. In order to fix this you could create a new database migration using the generate method. The foreign key would live in the Cohort model. It would be called cohort:id
```

Comment:
Nice clarification in the researched answer. You are correct that the foreign key comes from the cohort model, but it would be in the student's model.